.azure-pipelines/azure-pipelines-windows.yml

```diff
@@ -138,6 +138,7 @@ jobs:
                         psi4/label/dev::libint2=*=h2e52968_4 ^
                         conda-forge::qcelemental=0.25.0 ^
                         conda-forge::qcengine=0.23.0 ^
+                        conda-forge::optking=0.2.0 ^
```

Comment:
if you only put a version here b/c qcel/qcng had versions, then there's no need for it. easier to debug and update optking upstream if it's floating.

Reply:
That is exactly why I did :)